buildpacks/maven/src/framework.rs

```diff
@@ -97,12 +97,14 @@ pub(crate) fn default_app_process<P: AsRef<Path>>(
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Instead of disabling the lint here, wouldn't the better option be to take it's advice and drop the inner error if it's not being used anywhere?

Reply:
It is being used, but only implicitly. The `Debug` implementation is omitted from the dead code check - but we use that to display the inner error should this error occur.